filebeat/prospector/config.go

```diff
@@ -43,6 +44,7 @@ type prospectorConfig struct {
 	Pipeline             string             `config:"pipeline"`
 	Module               string             `config:"_module_name"`  // hidden option to set the module name
 	Fileset              string             `config:"_fileset_name"` // hidden option to set the fileset name
+	Filters              processors.PluginConfig `config:"filters"`
```

Comment:
Probably we should call it processors, as all of them will be available. It's already causing confusion on the metricbeat side ...